src/components/EditProduct.tsx

```diff
@@ -13,22 +13,20 @@
  *
  */
 
-import { EuiButton, EuiPanel } from "@elastic/eui";
-import ConfirmationDialogContext, {
-    ConfirmDialogActions,
-    ShowConfirmDialogType,
-} from "contextProviders/ConfirmationDialogProvider";
+import { EuiButton, EuiFlexGroup, EuiFlexItem, EuiPanel } from "@elastic/eui";
+import ConfirmationDialogContext from "contextProviders/ConfirmationDialogProvider";
 import { isDate } from "date-fns";
 import { formDate, formInput, formSelect } from "forms/Builder";
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { FormattedMessage, WrappedComponentProps, injectIntl } from "react-intl";
 import { RouteComponentProps } from "react-router";
 import { SingleValue } from "react-select";
 import ApplicationContext from "utils/ApplicationContext";
 import { setFlash } from "utils/Flash";
-import { Option, ProductBlock, Product as ProductData, Workflow } from "utils/types";
+import { Option, Product as ProductData } from "utils/types";
 import { isEmpty, stop } from "utils/Utils";
 
+import { intl } from "../locale/i18n";
```

Comment:
`../` can be removed
```suggestion
import { intl } from "locale/i18n";
```

change it in the other files too